engine/Assets/Scripts/Camera/FreeCameraMode.cs

```diff
@@ -125,7 +125,7 @@ public class FreeCameraMode : ICameraMode
     }
     
     public void LateUpdate(CameraController cam) {
-        cam.GroundRenderer.material.SetVector("FOCUS_POINT", cam.transform.position);
+        cam.GroundRenderer.material.SetVector("_GridFocusPoint", cam.transform.position);
```

Comment:
This is also going to have to change in the driver station camera mode. Might be worth merging in dev? I'm not sure what the best way to handle merging PRs is

Reply:
It'll be handled when resolving conflicts